src/common/evaluation/QA/eval_depthmap_models/src/test_evaluate.py

```diff
@@ -0,0 +1,193 @@
+import sys
+import shutil
+from pathlib import Path
+
+import pandas as pd
+import pickle
+
+here = Path(__file__).resolve()
+
+sys.path.append(str(here.parents[4]))
+
+REPO_DIR = str(here.parents[6].absolute())
+
+from evaluation.QA.eval_depthmap_models.src.evaluate import copy_dir, prepare_sample_dataset  # noqa: E402
+from evaluation.QA.eval_depthmap_models.src.evaluate import tf_load_pickle  # noqa: E402
+
+
+def test_copy_dir():
+    common_dir_path = Path(REPO_DIR + "/src/common")
+    temp_common_dir = Path(here.parent / "temp_common")
+    copy_dir(src=common_dir_path, tgt=temp_common_dir, glob_pattern='*/*.py', should_touch_init=True)
+
+    assert temp_common_dir.exists()
```

Comment:
This is more specific and therefore tests even more strictly

```suggestion
    assert temp_common_dir.is_dir()
```